packages/host/app/components/card-catalog-modal.gts

```diff
@@ -49,7 +49,7 @@ export default class CardCatalogModal extends Component<Signature> {
             <button
               {{on 'click' (fn this.pick undefined)}}
               class='dialog-box__close'
-            >x</button>
+            >⨯</button>
```

Comment:
this is a pet peeve of mine `x` is not the same as `⨯`

Reply:
Nice peeve. There is another one to be aware of, the ellipsis. I.e. `…` (the unicode ellipsis) vs `...` (just typing three dots)